services/ui-src/src/components/reports/ReportProvider.test.tsx

```diff
@@ -102,7 +107,7 @@ describe("ReportProvider", () => {
     const button = screen.getByText("Create");
     await userEvent.click(button);
     expect(screen.getByTestId("errorMessage")).toHaveTextContent(
-      /Report could not be updated/
+      /Something went wrong on our end. Refresh your screen and try again./
```

Comment:
This text is in a lot of files. If we need to update this again it may be a little tedious. Could we mock this error message somewhere and import it into whatever tests need it?